integration_tests/src/main/python/string_test.py

```diff
@@ -314,23 +314,12 @@ def test_length():
                 'CHAR_LENGTH(a)',
                 'CHARACTER_LENGTH(a)'))
 
-# Once the xfail is fixed this can replace test_initcap_space
-@incompat
-@pytest.mark.xfail(reason='https://github.com/NVIDIA/spark-rapids/issues/120')
 def test_initcap():
     # Because we don't use the same unicode version we need to limit
     # the charicter set to something more reasonable
     # upper and lower should cover the corner cases, this is mostly to
     # see if there are issues with spaces
-    gen = mk_str_gen('([aAbB]{0,5}[ \r\n\t]{1,2}){1,5}')
-    assert_gpu_and_cpu_are_equal_collect(
-            lambda spark: unary_op_df(spark, gen).select(
-                f.initcap(f.col('a'))))
-
-@incompat
-def test_initcap_space():
-    # we see a lot more space delim
-    gen = StringGen('([aAbB]{0,5}[ ]{1,2}){1,5}')
+    gen = mk_str_gen('([aAbB1357_@%-ȺéŉŸ]{0,16}[ \r\n\t]{1,2}){1,5}')
```

Comment:
Related to the other comment about incompat, the fact that we need to carefully limit the input character set shows that this isn't truly a compatible operation.

Reply:
Yes, may I know any issue for this change ?
I thought we should cover at least some of the characters mentioned in the bug https://github.com/rapidsai/cudf/issues/3132 .